core/scripts/chaincli/handler/keeper.go

```diff
@@ -347,8 +347,8 @@ func (k *Keeper) keepers() ([]common.Address, []common.Address) {
 
 // createKeeperJobOnExistingNode connect to existing node to create keeper job
 func (k *Keeper) createKeeperJobOnExistingNode(urlStr, email, password, registryAddr, nodeAddr string) error {
-	lggr, close := logger.NewLogger()
-	defer close()
+	lggr, closeLogger := logger.NewLogger()
+	defer closeLogger()
```

Comment:
nit/ :shrug: 
```suggestion
	lggr, closeLggr := logger.NewLogger()
	defer closeLggr()
```

Reply:
I usually favor local style for readability. Particularly when the two vars are so closely related - e.g. in this case someone might wonder if `lggr != logger` is by design to distinguish the two from one another and question what the close func is doing.